sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List, Text, Dict, Any
+# from typing import List, Text, Dict, Any
 
 
-def fix_big_data_passing(workflow: List[Dict[Text, Any]]) -> List[Dict[Text, Any]]:  # Tekton change signature
+def fix_big_data_passing(workflow: dict) -> dict:
```

Comment:
Hi, @ckadner , could you please don't change the type input parma type for fix_big_data_passing? as I already hanled it in another PR with List type, and hanled pipelinerun together. Thanks ckadner very much!